pkg/occlient/occlient.go

```diff
@@ -2629,12 +2606,22 @@ func makeTar(srcPath, destPath string, writer io.Writer, files []string) error {
 	srcPath = path.Clean(srcPath)
 	destPath = path.Clean(destPath)
 
+	glog.V(4).Infof("makeTar: srcPath: %s, destPath: %s, files: %+v", srcPath, destPath, files)
 	if len(files) != 0 {
 		//watchTar
 		for _, fileName := range files {
 			if checkFileExist(fileName) {
 				// The file could be a regular file or even a folder, so use recursiveTar which handles symlinks, regular files and folders
-				return recursiveTar(path.Dir(srcPath), path.Base(srcPath), path.Dir(destPath), path.Base(destPath), tarWriter)
+				srcFile, err := filepath.Rel(srcPath, fileName)
```

Comment:
Need to add an updated comment here on what's happening. The above comment is no longer relevant.

Reply:
Ack